### PR TITLE
Add perameters for grades in amy.json, complete grades page draft

### PR DESCRIPTION
**Pull request by bndnio**

### CODE REVIEW

File: templates/records/verification.hbs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <th class="col-md-8"></th>
+                <th class="col-md-2"></th>
+                <th class="col-md-2"></th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>{{#each user.records.grades}}
+                <td>{{semester}}</td>
+                <td>
+                    <a href="#" class="btn btn-info">Download <span class="glyphicon glyphicon-download"></span></a>
```
Comment (hwkr) [inline templates/records/verification.hbs:23]: Use font awesome instead

File: templates/records/transcripts.hbs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        </div>
+    </div>
+
+    <!--Official Transcript Column-->
+    <div class="col-md-4">
+        <h2>Official Transcript</h2>
+        <p>Some Text</p>
+
+        <div class="panel panel-default">
+            <div class="panel-body">
```
Comment (hwkr) [inline templates/records/transcripts.hbs:47]: This could be extracted into a partial

File: templates/records/transcripts.hbs [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                {{#each user.records.grades}}
+                    <h3>{{semester}}</h3>
+                    <p><span class="text-muted">Semester GPA </span><span class="badge">{{semester_GPA}}</span></p>
+                    <!-- Grade Table -->
+                    <table class="table">
+                        <thead>
+                        <tr>
+                            <th>Course</th>
```
Comment (hwkr) [inline templates/records/transcripts.hbs:23]: Could these strings go in globals?

File: templates/records/verification.hbs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+<div style="min-height: 60vh;">
+    {{! This is the starter for a section page
+        -------------------------------------------------------
+        - The tag at the top brings in the default portions
+        - Data comes in from JSON files, please put all data/strings in the JSON files
+        - Keep the data organized, if you're adding financial data put it under "finances"
+    }}
+    <div class="col-md-12">
+        <h4><b>Proof of Enrollment</b></h4>
+        <table class="table table-striped">
```
Comment (hwkr) [inline templates/records/verification.hbs:11]: Does this need to be a table? Maybe one of these? http://getbootstrap.com/components/#panels-tables

File: templates/records/verification.hbs [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            <thead>
```
Comment (hwkr) [inline templates/records/verification.hbs:12]: Do we need a header?

Comment (bndnio) [inline templates/records/verification.hbs:12]: It just helps by sizing the columns, looks a bit better on a smaller screen (if I change the sizing boundary) is all. Is there another/better way to do that?

File: templates/records/transcripts.hbs (hunk 3 above)
Comment (bndnio) [inline templates/records/transcripts.hbs:23]: What strings? Semester GPA and Courses?

Comment (hwkr) [inline templates/records/transcripts.hbs:23]: Course, percent, etc, also those

File: templates/records/verification.hbs (hunk 5 above)
Comment (hwkr) [inline templates/records/verification.hbs:12]: I'm not sure

**Pull request merged**